pkg/codegen/testing/test/testdata/azure-native-nested-types/nodejs/types/documentdb/input.ts

```diff
@@ -0,0 +1,7 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
```

Comment:
Special case for empty file to not emit empty modules? WDYT?

Reply:
That might be easier in a follow-on PR. Right now there are a lot of edge cases I'm coming up against regarding this same pattern. Before we merge, I can file an issue to handle this as a special case.
Suppose we implement a special case to omit an empty file. If either input or output module is empty, then the index file (which is generated separately) has to know that too, so that it doesn't generate an import for a file that isn't there.

Reply:
Actually, I suppose I can't open the issue until after this is merged, otherwise I'd be opening an issue about a problem that doesn't exist yet lol